mlir/tools/mlir-tblgen/EnumsGen.cpp

```diff
@@ -128,8 +143,95 @@ namespace llvm {
 inline ::llvm::raw_ostream &operator<<(::llvm::raw_ostream &p, {0} value) {{
   auto valueStr = stringifyEnum(value);
 )";
-  os << formatv(parsedAndPrinterStart, qualName, cppNamespace,
-                enumAttr.getSummary());
+
+  const char *parsedAndPrinterStartUnquotedBitEnum = R"(
+  namespace mlir {
+  template <typename T, typename>
+  struct FieldParser;
+
+  template<>
+  struct FieldParser<{0}, {0}> {{
```

Comment:
How was this handled before? I expected that this is would replace another `FieldParser` implementation.

Reply:
There's a FieldParser implementation above that's the old-style bit enum. It's handled by always quoting bit enum values that have more than one bit set.

Usually, these FieldParser values were very much unused, as there are separate parser implementatinos both in `EnumParameter` and in other bits of `mlir-tblgen` that mean this code was basically never called.